### DISCUSSION TITLE
Edge Box RPi 200 Getting Started with Node Red | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1922] [Q&A]
# Edge Box RPi 200 Getting Started with Node Red | Seeed Studio Wiki

Edge Box RPi 200 Getting Started with Node Red

https://wiki.seeedstudio.com/Edge-Box-Getting-Started-with-Node-Red/

**Comment by Madtechnician** [comment]
Does NOT work!!
 Install Node 20.18.1-1nodesource1   \u271\u2718   Bad install:  Node.js missing  Npm missing - Exit

Chinese download mirrors??
Antoher edgebox locked me out!! 
pi@raspberrypi:~ $ sudo apt-get update
sudo: account validation failure, is your account locked?

And not possible to install new OS???
